starboard/shared/starboard/application.h

```diff
@@ -362,6 +362,10 @@ class Application {
     command_line_.reset(new CommandLine(argc, argv));
   }
 
+  void SetCommandLine(std::unique_ptr<CommandLine> command_line) {
```

Comment:
How is this used in this PR?

Reply:
It's used on line 63 of starboard/android/shared/application_android.cc in this PR. You'd want to expand the file(I don't know why github is collapsing it... It's not a removal)